tests/test_salesforce_lookback_window.py

```diff
@@ -0,0 +1,108 @@
+from datetime import datetime, timedelta
+from tap_tester import connections, runner, menagerie
+from base import SalesforceBaseTest
+
+class SalesforceLookbackWindow(SalesforceBaseTest):
+
+    # subtract the desired amount of seconds form the date and return
+    def timedelta_formatted(self, dtime, seconds=0):
```

Comment:
It would be better if this method took in the format as a parameter so that it's usage is explicit. We should never manipulate results in the tests without being explicit, else we may cover up a bug.

Reply:
Added format param in the function.